### PR TITLE
Fix `profile-hotkey`

### LINKED ISSUE #3720: `profile-hotkey` is broken since the dropdown is lazy loaded
Should we not attach it to the button and just make our own shortcut?

### PR DESCRIPTION
**Pull request by yakov116**
1. LINKED ISSUES:
   fixes #3720 

2. TEST URLS:
 every where including gists

3. SCREENSHOT:
  none

### CODE REVIEW

File: source/features/profile-hotkey.tsx [hunk 1]
```diff
@@ -1,10 +1,13 @@
+import React from 'dom-chef';
 import select from 'select-dom';
 import onetime from 'onetime';
 
 import features from '.';
 
 function init(): void {
-	select('a[data-ga-click$="text:your profile"]')!.dataset.hotkey = 'g m';
+	// This exists on all pages even gists
+	const {origin} = new URL(select('.js-notification-shelf-include-fragment')!.dataset.baseSrc!);
```
Comment (fregante) [inline source/features/profile-hotkey.tsx:9]: What about `isEnterprise() ? location.origin : 'https://github.com'`

File: source/features/profile-hotkey.tsx [hunk 2]
```diff
@@ -1,10 +1,13 @@
...
+	document.body.append(<a className="d-none" data-hotkey="g m" href={origin + '/profile'}/>);
```
Comment (fregante) [inline source/features/profile-hotkey.tsx:10]: Prefer the `hidden` attribute

File: source/features/profile-hotkey.tsx (hunk 1 above)
Comment (yakov116) [reply source/features/profile-hotkey.tsx:9]: Gists

**Comment by fregante** [comment]
🐛 👍

**Pull request merged**